### PR TITLE
fix(tree): replace keyCode

### PR DESCRIPTION
**Pull request by Caaalabash**
+ replace `keyCode`  in `el-tree` component
+ remove unused type `ComponentInternalInstance` in `el-step`
+ some code style.

### CODE REVIEW

File: packages/utils/vnode.ts [hunk 1]
```diff
@@ -55,11 +55,7 @@ export const getFirstValidNode = (
   maxDepth = 3,
 ): ReturnType<typeof getChildren> => {
   if (Array.isArray(nodes)) {
-    for (let i = 0; i < nodes.length; i++) {
-      const node = nodes[i] as VNode
-      const child = getChildren(node as VNode, maxDepth)
-      return child
-    }
```
Comment (JeremyWuuuuu) [inline packages/utils/vnode.ts:62]: What error have you encountered?

File: packages/utils/aria.ts [hunk 2]
```diff
@@ -120,9 +120,9 @@ const Utils = {
    */
   focusFirstDescendant: function(element: HTMLElement): boolean {
     for (let i = 0; i < element.childNodes.length; i++) {
-      const child = element.childNodes[i]
+      const child = element.childNodes[i] as HTMLElement
       if (
-        attemptFocus(child as HTMLElement) ||
```
Comment (JeremyWuuuuu) [inline packages/utils/aria.ts:125]: what reason did you have on changing this?

**Review by jw-foss — COMMENTED** [review]
The rest LGTM, I need answers for the two comments

File: packages/utils/vnode.ts (hunk 1 above)
Comment (Caaalabash) [reply packages/utils/vnode.ts:62]:
````
    for (let i = 0; i < nodes.length; i++) {
      const node = nodes[i] as VNode
      const child = getChildren(node as VNode, maxDepth)
      return child
    }
````

only take the first node in nodes

File: packages/utils/aria.ts (hunk 2 above)
Comment (Caaalabash) [reply packages/utils/aria.ts:125]:
````
if (attemptFocus(child as HTMLElement) ||  this.focusLastDescendant(child) ) {
   return true
}
````

`focusLastDescendant` got warning: `Argument type ChildNode is not assignable to parameter type HTMLElement `

so change to 

````
const child = element.childNodes[i] as HTMLElement
````

**Review by jw-foss — APPROVED** [review]

**Pull request merged**